utility/BUILD.bazel

```diff
@@ -1,42 +1,40 @@
-# TODO: Rename file to 'BUILD' post-Gradle. See https://github.com/oppia/oppia-android/issues/1532.
+# TODO(#1532): Rename file to 'BUILD' post-Gradle.
+'''
+This library contains utilities that all other modules, minus model, depend on.
+It also contains Robolectric and JUnit tests for it's utilities.
+In Bazel, Kotlin source files are built into a library using the kt_android_library() rule.
+Kotlin test files must be built in their own kt_android_library() rule and added as a dependency to
+an android_local_test() rule which configures instrumentation tests in Bazel.
+'''
+
 load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
 load("@rules_jvm_external//:defs.bzl", "artifact")
 load("@dagger//:workspace_defs.bzl", "dagger_rules")
 
-'''A kt_android_library() rule that builds utility module kotlin source files into a library.'''
+# A kt_android_library() rule that builds utility module kotlin source files into a library.
```

Comment:
We should describe the library itself, eg:

``Library for general-purpose utilities.``

Reply:
Done! Should I add single line comments for each rule in a BUILD file?

Reply:
Thanks! It's up to you--if you think there's value in people knowing what does/should go in a library, then adding a comment is a good idea.

I generally think that the top-level file comment for the package is more valuable, though.

Reply:
Okay! Sounds good.